index.html

```diff
@@ -61,12 +61,14 @@
             <td><input type="button" value="1" onclick="dis('1')"/> </td> 
             <td><input type="button" value="2" onclick="dis('2')"/> </td> 
             <td><input type="button" value="3" onclick="dis('3')"/> </td> 
+            <td><input type="button" value="/" onclick="dis('/')"/> </td> 
```

Comment:
This is great but don't forget all the homework was looking for was 2 fields. That would have been 2 input boxes.